configs/eslint.json

```diff
@@ -50,5 +50,7 @@
     }
   },
   "scrap_start_urls": false,
+  "js_render": true,
+  "use_anchors": true,
```

Comment:
```suggestion
```

This setting's name is misleading. It lets know the scrapper that it should crawl pages with different anchors as different page. It is required for some website where the anchors define the routing such as https://nsfi.github.io/ppfish-components/#/home

We always retrieve the closest anchor from an element extracted. You can remove this parameter